bin/start-vm

```diff
@@ -255,6 +256,7 @@ set_binaries () {
         bin_stat="gstat"
         bin_sed="gsed"
         bin_uuid="/usr/bin/uuidgen"
+        bin_realpath="grealpath"
```

Comment:
the `g` prefix is macos/homebrew specific, on linux the binary is called `realpath`